### ISSUE TITLE
zrobot connect command seems to use hardcoded port

### ISSUE THREAD

**Issue by chrisvdg** [Jumpscale/0-robot#56]
Running a zrobot by default uses port 6600. When using the `zrobot connect` command to add the robot errors with (notice port in the error message) :
```bash
root@Jon:~/0-robot# zrobot robot connect main http://localhost:6600
[Mon05 11:56] - ConfigManager.py :108 :j - INFO - found jsconfig dir in: /opt/code/github/jumpscale/core9
service running at http://localhost:6000 doesn't seems to be a valid 0-robot
```

Running the zrobot on port 6000 does work fine.

**Comment by katia-e** [comment]
``` bash
zrobot robot connect main http://10.101.163.239:6600
[Mon05 12:32] - ConfigManager.py    :108 :j                              - INFO     - found jsconfig dir in: /opt/code/github/jumpscale/core9
service running at http://localhost:6000 doesn't seems to be a valid 0-robot
```

Presumably, the full address is hardcoded.

**Comment by chrisvdg** [comment]
After some testing I think this issue is reproduced by:

- running a zrobot on a certain port
- connect to it
- stop the zrobot
- run it again on another port
- connect to the new port

The fix currently in pr #62 does not fix this

Maybe there should also be the ability to disconnect zrobots?
